app/services/owner_classifier.rb

```diff
@@ -4,7 +4,27 @@ def initialize(id_and_type)
   end
 
   def user_or_usergroup
```

Comment:
I suspect that if you handle `nil` that the test failures will (mostly) disappear. Because AFAIK `Setting[:host_owner]` can be `nil` and that should be handled.
```suggestion
  def user_or_usergroup
    return nil unless @id_and_type

```

Reply:
Thank you for this suggestion!